tests/conftest.py

```diff
@@ -35,7 +35,9 @@ def app():
 
 @pytest.fixture
 def testapp(app):
-    return TestApp(app)
+    _app = TestApp(app)
+    _app.authorization = ('Basic', ('username', 'password'))
```

Comment:
I'm not sure I like the default being the testapp is authorized. I may just need more :coffee:  though.